docs/api/wallet-rpc.yaml

```diff
@@ -610,11 +610,69 @@ components:
       required:
         - destinations
       properties:
-        destinations:
+        destination_addresses:
           type: array
           items:
             type: string
             example: "bcrt1qujp2x2fv437493sm25gfjycns7d39exjnpptzw"
+        tumbler_options:
```

Comment:
I added all tumbler specific [options](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/jmclient/jmclient/cli_options.py#L268) from the `tumbler.py` script here. Not sure if it makes sense to document all of them here. Let me know what you think.

Reply:
I don't think we have to go nuts in terms of documentation straight away. I'm happy to add the relevant at a later point, especially considering that we have talked about the possibility of moving the tumbler default values to the general configuration in a future refactor.

Reply:
On this, (I recently looked at it, probably the last of these PRs) `maxcreatetx` and `maxbroadcasts` are no longer used, they are a leftover from an older version of the Joinmarket code.

I'll PR removing them later, but you can drop them here. The rest are all used.

Reply:
Oh, sorry, missed one: `donateamount` likewise has been unused for a long time; remove that too.

Reply:
Removed `maxcreatetx`, `maxbroadcasts`, and `donateamount`.